docs/ldml/tr35-keyboards.md

```diff
@@ -2347,6 +2619,21 @@ Another partial example allows a keyboard implementation to prevent people typin
 </transformGroup>
 ```
 
+#### Reorder and Markers
+
+The reorders do not themselves interact with markers, that is, markers may not be matched by a `reorder` element. However, if a character preceded by one or more markers is reordered due to a `reorder` element, those markers will also move with the characters as with the `transform` elements.
```

Comment:
"The reorders" sounds strange. Do you mean the processing according to reorder elements? Maybe "reorder processing" would be clearer, if that is what you mean